docs/modules/reference/pages/configuration/file-editor.adoc

```diff
@@ -1,66 +1,68 @@
-[[file-editor]]
-= Manage Configuration Files in the UI
-
-The File Editor, available in the new UI, lets users view and edit configuration files (`.xml`, `.cfg`, `.drl`, etc.) with the OpenNMS UI instead of having to edit them directly in the file system (for example in the `/etc` directory).
 
-NOTE: Making changes to configuration files will affect how OpenNMS operates and incorrectly editing a file could result in features no longer working or not working as intended.
+[[file-editor]]
+= File Editor
 
-To access the File Editor, go to the new UI, for example by clicking on "UI Preview", then navigating to "File Editor".
+The File Editor page, available in the new web UI, lets you view and edit configuration files (`.xml`, `.cfg`, `.drl`, and so on) in the {page-component-title} UI instead of having to edit them in your local file system (for example, by locating the file in your `/etc` directory and editing it with a text editor).
+To access the file editor, log in to the web UI and click menu:Info[File Editor] in the top menu bar.
 
-== View Files
+NOTE: Making changes to configuration files will affect the operation of {page-component-title}.
+Incorrectly editing a file could result in features no longer working, or not working as intended.
 
-After navigating to the File Editor, the UI displays a tree view of the OpenNMS `/etc` directory.
+== View files
 
-From here you can:
+By default, the File Editor page displays a tree view of your OpenNMS `/etc` directory.
+From this page, you can do the following:
 
-- navigate to any file
-- expand and collapse subtrees
-- search for any part of a file or directory name
-- view and edit files
+* Navigate to and open any file in your OpenNMS directory.
+* Expand and collapse subtrees (subdirectories).
+* Search for a directory name.
+* Search for any part of a file.
+* View and edit files.
 
-.File Editor main screen
+.File Editor page displaying the directory tree and an open configuration file
 image::configuration/file-editor-main.png[]
```

Comment:
replace this image with one that is cropped to eliminate the left menu and main title bar, similar to the logs screen earlier.